.travis.yml

```diff
@@ -39,10 +57,15 @@ before_install:
 
 install:
     - ./autogen.sh
+    # Build rdma-core because ubuntu trusty doesn't have a sufficiently new version of ibverbs/rdma-core
     # Build verbs only in linux as OS X doesn't have verbs support
-    - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
-        LIBRARY_CONFIGURE_ARGS="$LIBFABRIC_CONFIGURE_ARGS --enable-usnic
-        --enable-verbs --enable-mlx=$HOME/mlx";
+    - if [[ "$TRAVIS_OS_NAME" == "linux" ]] ; then
+        RDMA_CORE_BRANCH=v20 ;
```

Comment:
Thinking about this, I think we want the oldest version that's required.  Otherwise, we risk introducing a change that will only work with a newer version of the library and won't detect the breakage.

Reply:
Sounds good. I'll investigate which tag we can use.

Reply:
You may want to look at v13, I think v12 does not have the required RDMA CM routines.

Reply:
I tested this on this branch and on the feature/xrc branch -- with v13 rdma-core and travis passes.

Reply:
https://travis-ci.com/jswaro/libfabric/jobs/157094370